src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="https://www.moyo.ua/"
+        class="logo"
+      >
+        <img
+          src="images/moyo_1.png"
```

Comment:
You already have this image
```suggestion
          src="images/moyo.png"
```